docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -14,289 +14,324 @@ need to implement it on your own. Don't worry, it's easy!
+``@linter``:
 
 ::
 
-    def run(self, filename, file):
+    from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
-        return self.lint(filename)
+    @linter(...
+            severity_map={'W': RESULT_SEVERITY.NORMAL,
+                          'F': RESULT_SEVERITY.MAJOR,
+                          'E': RESULT_SEVERITY.MAJOR,
+                          'C': RESULT_SEVERITY.NORMAL,
+                          'R': RESULT_SEVERITY.NORMAL,
+                          'I': RESULT_SEVERITY.INFO},
+            ...)
 
-And that should be enough. The lint() method of the Lint class will do the
-rest for us.
+``coalib.results.RESULT_SEVERITY`` contains three different values, ``Info``,
+``Warning`` and ``Error`` you can use.
 
 We can test our bear like this
 
 ::
 
-    $ coala --bear-dirs=. --bears=HTMLTutorialBear --files=sample.html
+    $ coala --bear-dirs=. --bears=PylintTutorialBear --files=sample.py
 
 .. note::
 
     In order for the above command to work we should have 2 files in
-    our current dir: ``HTMLTutorialBear.py`` and our ``sample.html``.
+    our current dir: ``PylintTutorialBear.py`` and our ``sample.py``.
     Naming is **very** important in *coala*. *coala* will look for bears
     by their **filename** and display them based on their
     **classname**.
 
-Adding Settings to our Bear
----------------------------
+Using the ``corrected`` Output Format
+-------------------------------------
 
-If we run
+This output format is very simple to use and doesn't require further setup from
+your side inside the bear:
 
 ::
 
-    $ html_lint.py -h
+    @linter(...
+            output_format='corrected')
+
+If your underlying tool generates a corrected file, the class automatically
+generates patches for the changes made and yields results accordingly.
 
-We can see that there is a ``--disable`` option which lets us disable some
-checks. Let's add that functionality to our bear.
+Adding Settings to our Bear
+---------------------------
 
-First of all we have to import the setting that we are going to use from
-coalib. Since ``--disable`` needs a comma separated list we can use a list
-to keep our options. For that we will import ``typed_list`` like so
+If we run
 
 ::
 
-    from coalib.settings.Setting import typed_list
+    $ pylint --help
 
-``typed_list(item_type)`` is a function that converts the given input
-(which the user will pass to the Bear as a setting) into a list of
-items and afterwards will apply a conversion to type ``item_type`` to each
-item in the list (you can also use basic types like ``int``, ``bool``, etc.
-see :doc:`Writing Bears <Writing_Bears>`)
-Next, we have to add our setting as a parameter for the ``run()`` method
-of our bear.
-We will give the param a sugestive name like ``htmllint_ignore``.
+We can see that there is a ``--rcfile`` option which lets us specify a
+configuration file for Pylint. Let's add that functionality to our bear.
```

Comment:
when thinking about it, as pylint allows to supply everything via command line, using `create_arguments` only is preferred imo.
Maybe we should do another tutorial for this topic? But then we need to do it for another tool and the user needs to reread stuff again...